input/contribute/developer-experience/the-pitch.md

```diff
@@ -7,4 +7,18 @@ Case in point - AngularJS was released, people adopted it, and then Version v2.0
 
 Internally the we debate whether ReactiveUI is or is not a framework, as at its core the project is essentially a bunch of extension methods for the Reactive Extensions.
 
-If you are sick of investing personal development energy in transitory frameworks whereby you must learn "how do I do $x with $framework specific way" knowing deep down in side that this knowledge has no value in the future; then please come and join us over at https://reactiveui.net/slack
\ No newline at end of file
+If you are sick of investing personal development energy in transitory frameworks whereby you must learn "how do I do $x with $framework specific way" knowing deep down in side that this knowledge has no value in the future; then please come and join us over at https://reactiveui.net/slack
+
+# Have you decided to adopt Rx?
+
+Have you solved a complex problem with Reactive Extensions in your code?  Did you implement a new feature using Rx and want to learn more about Reactive Programming?  Are you looking for a better way to deal with mutable state?  Have you heard of this horrible concept of Rx and want to know why its bad and you should stay away from it?  If the answer is yes to any of these questions, welcome to ReactiveUI.
+
+# What Rx can do for you?
+
+> ReactiveUI at its core is a set of extension methods for the Reactive Extensions
+
+ That means it isn't really an MVVM framework, but can be wired into MVVM programming to provide reactive functionality to your MVVM experience.  Your not learning another MVVM framework that will be obsolete in the future.  You are applying Reactive Programming, which exists across many popular languages.  The skills are transferable, because it's *all* reactive functional programming.  Buyer be warned! Rx is addictive.  Once you start reacting to behavior, you won't want to give explicit instructions.
```

Comment:
> That means it isn't really an MVVM framework

Close; ReactiveUI actually is an MVVM framework but it's so very different to others. We provide a few base classes that allow people to program in their platform of choice in a reactive manner.